tests/jupyterhub_api/SessionUtils.py

```diff
@@ -0,0 +1,65 @@
+import os
+import subprocess
+import requests
+import json
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+class CreateSession:
+
+    def __init__(self, port, users, path):
+        self.port = port
+        self.users = users
+        self.path = path
```

Comment:
Here the "path" parameter corresponds to the path to the jupyterhub_config file which recognises the "dummy_admin"